systemd/network-detection-check.timer

```diff
@@ -1,5 +1,5 @@
 [Unit]
-Description=Run the Packet Forwarder status check every 15 seconds.
+Description=Run the Server Detection status check every 15 seconds.
```

Comment:
@sunof-top `Run the Server Detection every 15 seconds` should be good.
and the filename should be rename to 'server-detection.service`